res/controllers/Traktor Kontrol S4 MK3.hid.xml

```diff
@@ -9,6 +9,615 @@
+                    <value label="Celeste">celeste</value>
+                    <value label="Fuscia">fuscia</value>
+                    <value label="Green">green</value>
+                    <value label="Lime">lime</value>
+                    <value label="Orange">orange</value>
+                    <value label="Purple">purple</value>
+                    <value label="Red">red</value>
+                    <value label="Salmon">salmon</value>
+                    <value label="Sky">sky</value>
+                    <value label="White">white</value>
+                    <value label="Yellow" default="true">yellow</value>
+                </option>
+                <option
+                    variable="deckD"
+                    type="enum"
+                    label="Deck D">
+                    <value label="Aqua">aqua</value>
+                    <value label="Azalea">azalea</value>
+                    <value label="Blue">blue</value>
+                    <value label="Celeste">celeste</value>
+                    <value label="Fuscia">fuscia</value>
+                    <value label="Green">green</value>
+                    <value label="Lime">lime</value>
+                    <value label="Orange">orange</value>
+                    <value label="Purple" default="true">purple</value>
+                    <value label="Red">red</value>
+                    <value label="Salmon">salmon</value>
+                    <value label="Sky">sky</value>
+                    <value label="White">white</value>
+                    <value label="Yellow">yellow</value>
+                </option>
+            </row>
+            <option
+                variable="keepLEDWithOneColorDimedWhenInactive"
+                type="boolean"
+                default="true"
+                label="Keep one-color LED dimmed instead of off when inactive">
+                <description>
+                    This will always ensure every button has some lighting which may help in dark environment, but may put off the lead color of the deck.
+                </description>
+            </option>
+            <option
+                variable="gridButtonBlinkOverBeat"
+                type="boolean"
+                default="false"
+                label="Make the GRID button blink over beat">
+                <description>
+                    This will Make the GRID button blink when the track is going over a detected beat. This can help adjusting the beat grid or BPM.
+                </description>
+            </option>
+            <option
+                variable="keepDeckSelectDimmed"
+                type="boolean"
+                default="true"
+                label="Keep both deck select buttons backlit">
+                <description>
+                    Keep both deck select buttons backlit and do not fully turn off the inactive deck button. If enabled, will keep the unseclected deck dimmed, otherwise it will fully turn it off.
```

Comment:
please run a spell check on this PR, there are a lot of words :)

```suggestion
                    Keep both deck select buttons backlit and do not fully turn off the inactive deck button. If enabled, will keep the unselected deck dimmed, otherwise it will fully turn it off.
```